x-pack/test/functional/apps/ml/anomaly_detection/single_metric_job.ts

```diff
@@ -201,7 +202,26 @@ export default function ({ getService }: FtrProviderContext) {
       await ml.api.assertDetectorResultsExist(jobId, 0);
     });
 
+    it('job cloning fails in the single metric wizard if a matching data view does not exist', async () => {
+      await ml.testExecution.logTestStep('delete data view used by job');
+      await ml.testResources.deleteIndexPatternByTitle('ft_farequote');
+
+      // Refresh page to ensure page has correct cache of data views
+      await browser.refresh();
+
+      await ml.testExecution.logTestStep(
+        'job cloning clicks the clone action and loads the single metric wizard'
+      );
+      await ml.jobTable.clickCloneJobActionWhenNoDataViewExists(jobId);
```

Comment:
The test step message is not 100% accurate. Maybe something like `job cloning clicks the clone action and displays an error toast`?

Reply:
Done in https://github.com/elastic/kibana/pull/117993/commits/b67a413b1d958a3c05428f18fa6218320ebd8a11.